compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionUtils.cpp

```diff
@@ -130,6 +130,12 @@ static bool isReductionBroadcastElementwise(OpOperand *operand) {
   // Check if the consumer is an elementwise with identity output indexing map.
   if (!isSimpleElementwise(consumer)) return false;
 
+  // When we have static shapes, we do extra checks for the type. For dynamic
+  // shape cases, we do not check the shape and do aggressive fusion with high
+  // optimism, which is the default approach we are pursuing now.
+  bool hasStaticShape =
```

Comment:
Nit: Move this closer to where it is used.

Reply:
Both if and else blocks just below use the variable and that's why it is outside.